templates/includes/highlights-gallery.html

```diff
@@ -29,7 +29,7 @@ <h2 class="highlight-gallery__heading">{{ highlight.image.title }}</h2>
                              height="{{ base_img.height }}"
                              width="{{ base_img.width }}"
                              class="highlight-gallery__image"
-                             {% if highlight.image.alt_text %}alt="{{ highlight.image.alt_text }}"{% endif %}/>
+                             {% if highlight.alt_text %}alt="{{ highlight.alt_text }}"{% endif %}/>
```

Comment:
`<img>` tags should ALWAYS have an `alt` attribute, even if the value is an empty string. So, we can ditch the conditional here.